flink-python/pyflink/table/table_environment.py

```diff
@@ -98,18 +99,25 @@ def __init__(self, j_tenv, serializer=PickleSerializer()):
         self._open()
 
     @staticmethod
-    def create(environment_settings: EnvironmentSettings) -> 'TableEnvironment':
```

Comment:
Also update the documentation in https://nightlies.apache.org/flink/flink-docs-master/docs/dev/python/python_config/.